### ISSUE TITLE
softmax in AdaptiveConv class

### ISSUE THREAD

**Issue by diweiqiang** [lxtGH/OctaveConv_pytorch#19]
首先非常感谢您的代码，让人受益匪浅，在这里提一点发现的可能的问题：在文件resnet_adaptiveconv.py的class AdaptiveConv中的代码行“weight = self.softmax(self.w)”，看起来直接对shape为torch.Size([3, 1, 64, 64])的self.w按您代码中如下softmax并没起作用，操作后weight仍然是全1的矩阵

**Comment by lxtGH** [comment]
self.w = nn.Parameter(torch.ones(3, 1, self.size[0], self.size[1])),  is a Parameter which can be trained with end2end training and use 1 just for initialization
